src/core/components/pin/gc.js

```diff
@@ -0,0 +1,153 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+const CID = require('cids')
+const base32 = require('base32.js')
+const parallel = require('async/parallel')
+const mapLimit = require('async/mapLimit')
+const { Key } = require('interface-datastore')
+const log = require('debug')('ipfs:gc')
+
+// Limit on the number of parallel block remove operations
+const BLOCK_RM_CONCURRENCY = 256
+const MFS_ROOT_DS_KEY = new Key('/local/filesroot')
```

Comment:
Can this be imported from `ipfs-mfs`? Would be nice if we didn't have to reach all the way in to https://github.com/ipfs/js-ipfs-mfs/blob/eb04d6d4cb9e3cfdb617ad29a271dc59af9dfd87/src/core/utils/constants.js#L13 cc @achingbrain

Reply:
Agreed :+1: